cmd/root.go

```diff
@@ -47,16 +45,6 @@ func Execute() {
 	cobra.CheckErr(rootCmd.Execute())
 }
 
```

Comment:
Isn't this still needed to support reading of value.yaml (and overriding it). for iter8 and controller?

Reply:
The linter said that this function was not used so I removed it but I think you are right

Reply:
This function was only used in `iter8 k launch`, not in `iter8 k run`.